docs/index.md

```diff
@@ -335,4 +335,4 @@ terraform {
 
 ## Project Support
 
-**Important:** Projects in the `databrickslabs` GitHub account, including the Databricks Terraform Provider, are not formally supported by Databricks. They are maintained by Databricks Field teams and provided as-is. There is no service level agreement (SLA). Databricks makes no guarantees of any kind. If you discover an issue with the provider, please file a GitHub Issue on the repo, and it will be reviewed by project maintainers as time permits.
+> ❗️ **Important:** Projects in the `databrickslabs` GitHub account, including the Databricks Terraform Provider, are not formally supported by Databricks. They are maintained by Databricks Field teams and provided as-is. There is no service level agreement (SLA). Databricks makes no guarantees of any kind. If you discover an issue with the provider, please file a GitHub Issue on the repo, and it will be reviewed by project maintainers as time permits.
```

Comment:
replace with registry-compatible notification box, though i'm not sure if this section must be highlighted that way. e.g. it sends the wrong messaging of "don't use it".

Reply:
Agreed, removed this.

Reply:
fixed.